components/community/Hero.tsx

```diff
@@ -0,0 +1,49 @@
+import orbitData from '../../config/orbitData.json';
+import Header from './Header';
+
+interface HeroProps {
+  className?: string;
+};
+
+/**
+ * @description This component displays Hero Section.
+ * @param {HeroProps} props - The props for Hero component.
+ * @param {string} props.className - Additional CSS classes for styling.
+ */
+export default function Hero({
+  // eslint-disable-next-line no-unused-vars, unused-imports/no-unused-vars
+  className = ''
```

Comment:
You can include className prop to add className to the top-level div below

Reply:
Done